SubEthaEdit-Mac/Source/PlainTextDocument.m

```diff
@@ -2680,7 +2693,14 @@ - (BOOL)readSEETextFromURL:(NSURL *)anURL properties:(NSDictionary *)aProperties
     [[self session] addContributors:contributors];
     // load text into dictionary
     NSMutableDictionary *storageRep = [dictRep objectForKey:@"TextStorage"];
-    NSString *string = [NSString stringWithContentsOfURL:[NSURL fileURLWithPath:[[anURL path] stringByAppendingPathComponent:@"plain.txt"]]  encoding:[dictRep objectForKey:@"AutosaveInformation"] ? NSUTF8StringEncoding : (NSStringEncoding)[[storageRep objectForKey:@"Encoding"] unsignedIntValue] error:outError];
+    
+    NSURL *textDataURL = [NSURL fileURLWithPath:[[anURL path] stringByAppendingPathComponent:@"plain.txt"]];
+    NSData *textData = [NSData dataWithContentsOfURL:textDataURL options:0 error:outError];
+    
+    if(!textData) return NO;
```

Comment:
style: space after `if`, and please no new if/else bodyes without curlys. prefarably on extra line too. e.g. 

```objective-c
if (!textData) {
  return NO;
}
```